src/Chapter06/Listing06.35C.DisablingRequiredObjectInitialization.cs

```diff
@@ -0,0 +1,69 @@
+#if NET7_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_34C
```

Comment:
File scoped namespace?

Reply:
I'll make all these changes in one swell foop at the end.